src/UniOneClient.php

```diff
@@ -94,11 +83,9 @@ public function setEndpoint(string $endpoint): void
      *
      * @return void
      */
-    public function setHttpClient(): void
+    public function setHttpClient(ClientInterface $client): void
```

Comment:
While we're touching those lines, let's return self in all `set` methods.

It'll allows us using chaining calls, for example:
`$client = new UniOneClient()->setEndpoint('some')->setApiKey('another');`